public/js/components/Editor.js

```diff
@@ -266,9 +265,12 @@ const Editor = React.createClass({
       enableCodeFolding: false,
       gutters: ["breakpoints"],
       value: " ",
-      extraKeys
+      extraKeys: {}
     });
 
+    // disables the default search shortcuts
+    this.editor._initShortcuts = () => {};
```

Comment:
Yeah, my initial change was to the editor.js for that reason, but I didn't want the headache of disabling it there, but not globally for all the tools. This will be better when our Editor Search Bar is global for all the tools `_shrug_`